bin/_test-helpers.sh

```diff
@@ -528,6 +529,12 @@ run_cluster-domain_test() {
   run_test "$test_directory/install_test.go" --cluster-domain='custom.domain'
 }
 
+# wrapper to implement external tests
+run_externalresources_test(){
```

Comment:
nit: How about calling this `external-resources` as we follow those semantics everywhere?